server/routes/charts.js

```diff
@@ -54,8 +54,10 @@ router.route('/charts/:chart_id').get(ensureAuthenticated, async (req, res) => {
       labels,
       studyTotals,
       filters,
+      chartOwner,
     } = await graphDataController(
       dataDb,
+      appDb,
```

Comment:
we don't use this in the graph data controller aside from getting the chart owner and returning it to use it here. What do you think about setting the `chartOwner` as a constant in this file instead?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/289/commits/7a41087bdcfbb553720a2350c19c7281011466a8